start.sh

```diff
@@ -31,11 +31,11 @@ FILE=composer.json
 if [ ! -f "$FILE" ]; then
   echo "No composer.json found."
   exit
-else
-  echo "I found composer.json. Let's begin downloading required libraries."
-  sleep 2
 fi
 
+echo "I found composer.json. Let's begin downloading required libraries."
+sleep 2
+
```

Comment:
`else` is not necessary here, execution is stopped by `exit`.

Reply:
That's true! Thanks for that.